jme3-core/src/com/jme3/gde/core/util/datatransfer/MeshDataFromOriginal.java

```diff
@@ -0,0 +1,96 @@
+package com.jme3.gde.core.util.datatransfer;
+
+import com.jme3.gde.core.scene.ApplicationLogHandler.LogLevel;
+import com.jme3.gde.core.util.SpatialUtil;
+import com.jme3.gde.core.util.TaggedSpatialFinder;
+import com.jme3.scene.Geometry;
+import com.jme3.scene.Node;
+import com.jme3.scene.SceneGraphVisitorAdapter;
+import com.jme3.scene.Spatial;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Copies mesh data from an updated external file to the spatial
+ */
+public class MeshDataFromOriginal implements SpatialDataTransferInterface {
```

Comment:
this name is not misleading (see the comment on TransformDataFromOriginal), but if you change it there, you also need to change it here, I guess.

Reply:
Consistency ftw!